commit 68f9f2521ea0f1f6d3cbef67a44d000aa23870fa
Merge: 273502a 8d80cbb
Author: ErikZalm <erik@vdzalm.eu>
Date:   Tue May 21 08:17:21 2013 -0700

    Merge pull request #487 from MStohn/fix-bad-lcd-commit-from-buildrob
    
    fix broken LCD commit #dfa549f268 from buildrob - GLCD/DOGM implementati...

diff --git a/Marlin/dogm_lcd_implementation.h b/Marlin/dogm_lcd_implementation.h
index c5192e7..c45a6a6 100644
--- a/Marlin/dogm_lcd_implementation.h
+++ b/Marlin/dogm_lcd_implementation.h
@@ -19,6 +19,19 @@
 * Implementation of the LCD display routines for a DOGM128 graphic display. These are common LCD 128x64 pixel graphic displays.
 **/
 
+ifdef ULTIPANEL
+#define BLEN_A 0
+#define BLEN_B 1
+#define BLEN_C 2
+#define EN_A (1<<BLEN_A)
+#define EN_B (1<<BLEN_B)
+#define EN_C (1<<BLEN_C)
+#define encrot0 0
+#define encrot1 2
+#define encrot2 3
+#define encrot3 1
+#define LCD_CLICKED (buttons&EN_C)
+#endif
 
 // CHANGE_DE begin ***
 #include <U8glib.h>	// DE_U8glib
